src/components/molecules/ImageInput/ImageCollectionInput.tsx

```diff
@@ -131,9 +131,11 @@ export const ImageCollectionInput: React.FC<ImageInputProps> = (props) => {
         value={imageUrlForPreview}
       />
       {error?.message && <span className="input-error">{error.message}</span>}
-      <div style={{ marginTop: 10, fontSize: "16px" }}>
-        {`Or choose one of our popular ${imageType}`}
-      </div>
+      {imageCollection.length ? (
+        <div style={{ marginTop: 10, fontSize: "16px" }}>
+          {`Or choose one of our popular ${imageType}`}
+        </div>
+      ) : null}
```

Comment:
Please, move the length to `hasImageCollections` and then use hasImageCollections && <div>..., not a ternary

Reply:
fixed in 65b0fee